Package.swift

```diff
@@ -17,6 +17,7 @@ import PackageDescription
 
 let package = Package(
     name: "async-http-client",
+    platforms: [.macOS(.v10_11), .iOS(.v9), .tvOS(.v9), .watchOS(.v2)],
```

Comment:
would this potentially confuse folks to think linux is not supported? maybe worth adding a comment?

Reply:
added a comment!

Reply:
I think we need to take this out. If we don't then _every dependency_ needs to mirror this stanza.